app/manifest.yml

```diff
@@ -46,6 +46,9 @@ permissions:
     fetch:
       client:
         - 'https://api-private.atlassian.com'
+        - 'https://app.launchdarkly.com'
+        - 'https://events.launchdarkly.com'
+        - 'https://clientstream.launchdarkly.com'
```

Comment:
We need to add all 3 of these so they don't get blocked during the requests to LD.